dotnet/src/Connectors/Connectors.AzureOpenAI/Settings/AzureOpenAITextToAudioExecutionSettings.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.SemanticKernel.Text;
+
+namespace Microsoft.SemanticKernel.Connectors.AzureOpenAI;
+
+/// <summary>
+/// Execution settings for Azure OpenAI text-to-audio request.
+/// </summary>
+[Experimental("SKEXP0001")]
+public sealed class AzureOpenAITextToAudioExecutionSettings : PromptExecutionSettings
+{
+    /// <summary>
+    /// The voice to use when generating the audio. Supported voices are alloy, echo, fable, onyx, nova, and shimmer.
+    /// </summary>
+    [JsonPropertyName("voice")]
+    public string Voice
+    {
+        get => this._voice;
+
+        set
+        {
+            this.ThrowIfFrozen();
+            this._voice = value;
+        }
+    }
+
+    /// <summary>
+    /// The format to audio in. Supported formats are mp3, opus, aac, and flac.
+    /// </summary>
+    [JsonPropertyName("response_format")]
+    public string ResponseFormat
+    {
+        get => this._responseFormat;
+
+        set
+        {
+            this.ThrowIfFrozen();
+            this._responseFormat = value;
+        }
+    }
+
+    /// <summary>
+    /// The speed of the generated audio. Select a value from 0.25 to 4.0. 1.0 is the default.
+    /// </summary>
+    [JsonPropertyName("speed")]
+    public float Speed
```

Comment:
This should be optional

Reply:
We'll update all properties of all new *PromptExecutionSettings classes in on go in scope of the issue - https://github.com/microsoft/semantic-kernel/issues/7111.